package.json

```diff
@@ -21,7 +21,6 @@
     "@theforeman/stories": "^10.0",
     "@theforeman/test": "^10.0",
     "@theforeman/vendor-dev": "^10.0",
-    "@testing-library/user-event": "^13.2.1",
```

Comment:
Is this intentional?

Reply:
I can try to delete it and see if it helps.

Reply:
I guess we need this package for JS tests

Reply:
It's written there twice (also on [line 17](https://github.com/theforeman/foreman_ansible/blob/ba8db81538f635f129f41b8a217736ee107c3664/package.json#L17)).